easyvista/help.md

```diff
@@ -0,0 +1,488 @@
+# Description
+
+EasyVista Service Manager platform supports even the most complex requirements, while bringing a new level of  simplicity, agility, and mobility required to make cloud based IT Service Management (ITSM) software easy to use and easy to deliver. Using the EasyVista plugin for Rapid7 InsightConnect, users can manage the creation, update, search and closure of incident, service request, problem or event tickets
+
+# Key Features
+
+* Create, close, update and search tickets
+
+# Requirements
+
+* EasyVista Service Manager username and password
+* EasyVista Service Manager server URL and account
+
+# Documentation
+
+## Setup
+
+The connection configuration accepts the following parameters:
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|account|integer|50004|True|Service Manager account used|None|50004|
+|client_login|credential_username_password|None|True|The EasyVista username and password for basic authentication API interaction|None|{"username":"user1", "password":"mypassword"}|
+|hostname|string|None|True|The hostname of your EasyVista server, e.g. https://example.easyvista.com|None|https://example.easyvista.com|
+
+Example input:
+
+```
+{
+  "account": 50004,
+  "client_login": {
+    "username": "user1",
+    "password": "mypassword"
+  },
+  "hostname": "https://example.easyvista.com"
+}
+```
+
+## Technical Details
+
+### Actions
+
+#### Search Tickets
+
+This action is used to search the EasyVista tickets.
```

Comment:
Can you include a link to the search filters with this? https://wiki.easyvista.com/xwiki/bin/view/Documentation/REST+API+-+Options+for+Fields#SearchFilterOptions